mgmtfn/dockplugin/netDriver.go

```diff
@@ -433,14 +435,27 @@ func GetDockerNetworkName(nwID string) (string, string, string, error) {
 			var tenantName, netName, serviceName string
 			names := strings.Split(nw.Name, ".")
 			if len(names) == 2 {
-				// has only network.tenant format
-				tenantName = names[1]
-				netName = names[0]
+				// determine if this is service.network on default tenant or network.tenant
+				_, err = netdGetNetwork(fmt.Sprintf("%s.%s", names[1], defaultTenantName))
+				if err == nil {
+					// This is service.network on default tenant
+					tenantName = defaultTenantName
+					netName = names[1]
+					serviceName = names[0]
+				} else {
+					// this is in network.tenant format
+					tenantName = names[1]
+					netName = names[0]
+				}
 			} else if len(names) == 3 {
 				// has service.network.tenant format
 				tenantName = names[2]
 				netName = names[1]
 				serviceName = names[0]
+			} else if len(names) == 1 {
+				// If only network is specified, use default tenant
+				tenantName = defaultTenantName
+				netName = names[0]
```

Comment:
need system tests for 'service.network/tenant', 'service.network', 'network', 'network/tenant' independently.